docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx

```diff
@@ -401,6 +415,489 @@ docker compose up
+
+<CodeExample>
+
+```bash
+transaction='
+{
+  "transaction": {
+    "id": '"$transaction_id"',
+    "status": "pending_external",
+    "external_transaction_id": "<id_of_external_transaction>",
+    "message": "Transfer is being verified"
+  }
+}
+'
+
+curl localhost:8080/transactions \
+    -X PATCH \
+    -H 'Content-Type: application/json' \
+    -d '{"records":['"$transaction"']}'
+```
+
+</CodeExample>
+
+#### Sending Stellar Transaction
+
+The next step would be sending Stellar transaction. First, let's change status to reflect the latest transaction state:
+
+<CodeExample>
+
+```bash
+transaction='
+{
+  "transaction": {
+    "id": '"$transaction_id"',
+    "status": "pending_anchor",
+    "message": "Transaction is ready to be fullfilled by the anchor"
+  }
+}
+'
+
+curl localhost:8080/transactions \
+    -X PATCH \
+    -H 'Content-Type: application/json' \
+    -d '{"records":['"$transaction"']}'
+```
+
+</CodeExample>
+
+Next, send a transaction in the Stellar network to fulfill user request. Currently, Anchor Platform doesn't offer such functionality. You may use a custodian service, or manage distribution account yourself. In latter case, you may want to use the [Stellar SDK](/docs/tools-and-sdks#sdk-library) on your server side to create, sign and submit transaction.
+
+:::info
+
+In the future, the Anchor Platform will also support making payments associated with deposit transactions to users on behalf of the business. This is another reason why you may want to use the Anchor Platform's event system.
+
+:::
+
+:::caution
+
+User account may not be ready to receive funds. You should check that account has established the [trustline](/docs/glossary#trustline). Otherwise, set transaction to status `pending_trust` to indicate that anchor is waiting for the user establishing the trustline.
+
+:::
+
+:::tip
+
+(Optional) You may set transaction status to `pending_stellar`, when it has been submitted to the network, but not yet included in the ledger. Due to the nature of Stellar network, transaction confirmation is almost instant. Some SDKs (Java) have blocking transaction submission method. After receiving success result from the submission method, transaction will already be completed.
+
+:::
+
+#### Completing Transaction
+
+Finally, after transaction has been sent and successfully submitted to the network, it's time to complete transaction:
+
+<CodeExample>
+
+```bash
+transaction='
+{
+  "id": '"$transaction_id"',
+  "status": "completed",
+  "message": "completed",
+  "stellar_transactions": [
+    {
+      "id": "<stellar_transaction_id>",
+      "payments": [
+        {
+          "id": "<payment_operation_id>",
+          "amount": {
+            "amount": "10",
+            "asset": "USDC:GBBD47IF6LWK7P7MDEVSCWR7DPUWV3NY3DTQEVFL4NAT4AQH3ZLLFLA5"
+          }
+        }
+      ]
```

Comment:
This should not be the schema for `stellar_transactions`, although I just checked the OpenAPI spec and it matches.

We have a list of Stellar transactions, each with a list of payments. I think we need to flatten this to just a list of payments that each have a transaction and operation ID. We can update the docs after we update the platform.

Reply:
Yes, it's the current schema. I don't think we would be able to flatten it, because it will mess up all structure in Java code

Reply:
I'm not sure I understand, but we can discuss it later, no need to block this PR.